install.rb

```diff
@@ -108,9 +108,12 @@ def do_man(man, strip = 'man/')
     FileUtils.makedirs(om, {:mode => 0755, :verbose => true})
     FileUtils.chmod(0755, om)
     FileUtils.install(mf, omf, {:mode => 0644, :preserve => true, :verbose => true})
-    gzip = %x{which gzip}
-    gzip.chomp!
-    %x{#{gzip} -f #{omf}}
+    # Solaris does not support gzipped man pages
+    unless Facter["os"].value["family"] == "Solaris"
```

Comment:
Hi @shawnferry - just changing the use of `Facter[...` to $operatingsystem should work. Feels a little less intuitive to me since $operatingsystem hasn't been defined yet but it's not worth the additional effort to substantially revise the options I suppose.